### ISSUE TITLE
broken package hdf4

### ISSUE THREAD

**Issue by starius** [mxe/mxe#720]
```
make[1]: Entering directory `/usr/lib/mxe'
uname -a
Linux sd-68075 3.2.0-4-amd64 #1 SMP Debian 3.2.63-2+deb7u1 x86_64 GNU/Linux
git show-branch --list --reflog=1
  [refs/heads/master@{0}] (2 hours ago) pull: Fast-forward
lsb_release -a 2>/dev/null || sw_vers 2>/dev/null || true
Distributor ID:	Debian
Description:	Debian GNU/Linux 7.8 (wheezy)
Release:	7.8
Codename:	wheezy
rm -rf   '/usr/lib/mxe/tmp-hdf4-i686-w64-mingw32.static'
mkdir -p '/usr/lib/mxe/tmp-hdf4-i686-w64-mingw32.static'
cd '/usr/lib/mxe/tmp-hdf4-i686-w64-mingw32.static' &&    tar xjf '/usr/lib/mxe/pkg/hdf-4.2.10.tar.bz2'
cd '/usr/lib/mxe/tmp-hdf4-i686-w64-mingw32.static/hdf-4.2.10'
(cd '/usr/lib/mxe/tmp-hdf4-i686-w64-mingw32.static/hdf-4.2.10' && patch -p1 -u) < ./src/hdf4-1-portability-fixes.patch
patching file configure.ac
Hunk #1 succeeded at 808 (offset 69 lines).
patching file hdf/src/hdfi.h
Hunk #1 succeeded at 589 (offset -276 lines).
patching file mfhdf/ncgen/Makefile.am
Hunk #1 succeeded at 26 (offset 3 lines).
cd '/usr/lib/mxe/tmp-hdf4-i686-w64-mingw32.static/hdf-4.2.10' && libtoolize --force
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, `bin'.
libtoolize: linking file `bin/ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIR, `m4'.
libtoolize: linking file `m4/libtool.m4'
libtoolize: linking file `m4/ltoptions.m4'
libtoolize: linking file `m4/ltsugar.m4'
libtoolize: linking file `m4/ltversion.m4'
libtoolize: linking file `m4/lt~obsolete.m4'
libtoolize: Consider adding `-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
cd '/usr/lib/mxe/tmp-hdf4-i686-w64-mingw32.static/hdf-4.2.10' && autoreconf --install
libtoolize: Consider adding `-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
automake: F77LINK was already defined in condition !HDF_BUILD_FORTRAN, which is included in condition TRUE ...
hdf/src/Makefile.am:48: ... `F77LINK' previously defined here
automake: F77LINK was already defined in condition !HDF_BUILD_FORTRAN, which is included in condition TRUE ...
mfhdf/libsrc/Makefile.am:56: ... `F77LINK' previously defined here
cd '/usr/lib/mxe/tmp-hdf4-i686-w64-mingw32.static/hdf-4.2.10' && ./configure --host='i686-w64-mingw32.static' --build='x86_64-unknown-linux-gnu' --prefix='/usr/lib/mxe/usr/i686-w64-mingw32.static'  --enable-static --disable-shared  --disable-fortran --disable-netcdf AR='i686-w64-mingw32.static-ar'  CPPFLAGS="-DH4_F77_FUNC\(name,NAME\)=NAME -DH4_BUILT_AS_STATIC_LIB=1"
configure: loading site script /usr/lib/mxe/usr/i686-w64-mingw32.static/share/config.site
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for i686-w64-mingw32.static-strip... i686-w64-mingw32.static-strip
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether to enable maintainer-specific portions of Makefiles... no
checking shell variables initial values... done
checking build system type... (cached) x86_64-unknown-linux-gnu
checking host system type... i686-w64-mingw32.static
checking if basename works... yes
checking if xargs works... yes
checking for config i686-w64-mingw32.static... no
checking for config i686-w64-mingw32.static... no
checking for config w64-mingw32.static... no
checking for config w64-mingw32.static... no
checking for config i686-mingw32.static... no
checking for config i686-mingw32.static... no
checking for config i686-w64... no
checking for config mingw32.static... no
checking for config mingw32.static... no
checking for config w64... no
checking for config i686... no
checking whether make sets $(MAKE)... (cached) yes
checking for i686-w64-mingw32.static-gcc... i686-w64-mingw32.static-gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.exe
checking for suffix of executables... .exe
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether i686-w64-mingw32.static-gcc accepts -g... yes
checking for i686-w64-mingw32.static-gcc option to accept ISO C89... none needed
checking for style of include used by make... GNU
checking dependency style of i686-w64-mingw32.static-gcc... gcc3
checking how to run the C preprocessor... i686-w64-mingw32.static-gcc -E
checking for i686-w64-mingw32.static-g++... i686-w64-mingw32.static-g++
checking whether we are using the GNU C++ compiler... yes
checking whether i686-w64-mingw32.static-g++ accepts -g... yes
checking dependency style of i686-w64-mingw32.static-g++... gcc3
checking whether ln -s works... yes
checking for ar... i686-w64-mingw32.static-ar
checking for bison... bison -y
checking for flex... flex
checking for diff... diff -w
checking for makeinfo... ${SHELL} /usr/lib/mxe/tmp-hdf4-i686-w64-mingw32.static/hdf-4.2.10/bin/missing --run makeinfo
checking for neqn... neqn
checking for tbl... tbl
checking how to print strings... printf
checking for a sed that does not truncate output... /bin/sed
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for fgrep... /bin/grep -F
checking for ld used by i686-w64-mingw32.static-gcc... /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-ld
checking if the linker (/usr/lib/mxe/usr/bin/i686-w64-mingw32.static-ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-nm -B
checking the name lister (/usr/lib/mxe/usr/bin/i686-w64-mingw32.static-nm -B) interface... BSD nm
checking the maximum length of command line arguments... 3458764513820540925
checking whether the shell understands some XSI constructs... yes
checking whether the shell understands "+="... yes
checking how to convert x86_64-unknown-linux-gnu file names to i686-w64-mingw32.static format... func_convert_file_nix_to_w32
checking how to convert x86_64-unknown-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-ld option to reload object files... -r
checking for i686-w64-mingw32.static-objdump... i686-w64-mingw32.static-objdump
checking how to recognize dependent libraries... file_magic ^x86 archive import|^x86 DLL
checking for i686-w64-mingw32.static-dlltool... i686-w64-mingw32.static-dlltool
checking how to associate runtime and link libraries... func_cygming_dll_for_implib
checking for i686-w64-mingw32.static-ar... (cached) i686-w64-mingw32.static-ar
checking for archiver @FILE support... @
checking for i686-w64-mingw32.static-strip... (cached) i686-w64-mingw32.static-strip
checking for i686-w64-mingw32.static-ranlib... i686-w64-mingw32.static-ranlib
checking command to parse /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-nm -B output from i686-w64-mingw32.static-gcc object... ok
checking for sysroot... no
checking for i686-w64-mingw32.static-mt... no
checking for mt... mt
configure: WARNING: using cross tools not prefixed with host triplet
checking if mt is a manifest tool... no
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if i686-w64-mingw32.static-gcc supports -fno-rtti -fno-exceptions... no
checking for i686-w64-mingw32.static-gcc option to produce PIC... -DDLL_EXPORT -DPIC
checking if i686-w64-mingw32.static-gcc PIC flag -DDLL_EXPORT -DPIC works... yes
checking if i686-w64-mingw32.static-gcc static flag -static works... yes
checking if i686-w64-mingw32.static-gcc supports -c -o file.o... yes
checking if i686-w64-mingw32.static-gcc supports -c -o file.o... (cached) yes
checking whether the i686-w64-mingw32.static-gcc linker (/usr/lib/mxe/usr/bin/i686-w64-mingw32.static-ld) supports shared libraries... yes
checking dynamic linker characteristics... Win32 ld.exe
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... no
checking whether to build static libraries... yes
checking how to run the C++ preprocessor... i686-w64-mingw32.static-g++ -E
checking for ld used by i686-w64-mingw32.static-g++... /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-ld
checking if the linker (/usr/lib/mxe/usr/bin/i686-w64-mingw32.static-ld) is GNU ld... yes
checking whether the i686-w64-mingw32.static-g++ linker (/usr/lib/mxe/usr/bin/i686-w64-mingw32.static-ld) supports shared libraries... yes
checking for i686-w64-mingw32.static-g++ option to produce PIC... -DDLL_EXPORT -DPIC
checking if i686-w64-mingw32.static-g++ PIC flag -DDLL_EXPORT -DPIC works... yes
checking if i686-w64-mingw32.static-g++ static flag -static works... yes
checking if i686-w64-mingw32.static-g++ supports -c -o file.o... yes
checking if i686-w64-mingw32.static-g++ supports -c -o file.o... (cached) yes
checking whether the i686-w64-mingw32.static-g++ linker (/usr/lib/mxe/usr/bin/i686-w64-mingw32.static-ld) supports shared libraries... yes
checking dynamic linker characteristics... Win32 ld.exe
checking how to hardcode library paths into programs... immediate
configure: creating ./config.lt
config.lt: creating libtool
checking if we should install only statically linked executables... no
checking zlib.h usability... yes
checking zlib.h presence... yes
checking for zlib.h... yes
checking for compress2 in -lz... yes
checking for compress2... yes
checking jpeglib.h usability... yes
checking jpeglib.h presence... yes
checking for jpeglib.h... yes
checking for jpeg_start_decompress in -ljpeg... yes
checking for szlib... suppressed
checking for xdr library support... yes
checking for ANSI C header files... (cached) yes
checking whether i686-w64-mingw32.static-gcc and cc understand -c and -o together... yes
checking how to run the C++ preprocessor... i686-w64-mingw32.static-g++ -E
checking size of int*... 4
checking whether byte ordering is bigendian... no
checking for build mode... production
checking for math library support... yes
checking for fork... no
checking for system... yes
checking for vfork... no
checking for wait... no
checking if deprecated public symbols are available... yes
configure: creating ./config.status
config.status: creating Makefile
config.status: creating libhdf4.settings
config.status: creating hdf/Makefile
config.status: creating hdf/examples/Makefile
config.status: creating hdf/fortran/Makefile
config.status: creating hdf/fortran/examples/Makefile
config.status: creating hdf/src/Makefile
config.status: creating hdf/test/Makefile
config.status: creating hdf/util/Makefile
config.status: creating hdf/util/h4cc
config.status: creating hdf/util/h4fc
config.status: creating hdf/util/h4redeploy
config.status: creating hdf/util/testutil.sh
config.status: creating man/Makefile
config.status: creating mfhdf/fortran/ftest.f
config.status: creating mfhdf/fortran/jackets.c
config.status: creating mfhdf/fortran/netcdf.inc
config.status: creating mfhdf/libsrc/netcdf.h
config.status: creating mfhdf/Makefile
config.status: creating mfhdf/dumper/Makefile
config.status: creating mfhdf/dumper/testhdp.sh
config.status: creating mfhdf/examples/Makefile
config.status: creating mfhdf/examples/testexamples.sh
config.status: creating mfhdf/fortran/Makefile
config.status: creating mfhdf/fortran/examples/Makefile
config.status: creating mfhdf/fortran/testfortran.sh
config.status: creating mfhdf/hdfimport/Makefile
config.status: creating mfhdf/hdfimport/testutil.sh
config.status: creating mfhdf/hdiff/Makefile
config.status: creating mfhdf/hdiff/testhdiff.sh
config.status: creating mfhdf/hrepack/Makefile
config.status: creating mfhdf/hrepack/hrepack_all.sh
config.status: creating mfhdf/libsrc/Makefile
config.status: creating mfhdf/ncdump/Makefile
config.status: creating mfhdf/ncdump/testncdump.sh
config.status: creating mfhdf/ncgen/Makefile
config.status: creating mfhdf/ncgen/testncgen.sh
config.status: creating mfhdf/nctest/Makefile
config.status: creating mfhdf/test/Makefile
config.status: creating mfhdf/test/testmfhdf.sh
config.status: creating mfhdf/xdr/Makefile
config.status: creating hdf/src/h4config.h
config.status: executing h4config commands
modifying hdf/src/h4config.h
Post process libhdf4.settings
config.status: executing depfiles commands
config.status: executing libtool commands
Renaming "mfhdf/libsrc/netcdf.h" to "mfhdf/libsrc/hdf4_netcdf.h" .
	    SUMMARY OF THE HDF4 CONFIGURATION
	    =================================

General Information:
-------------------
		   HDF4 Version: 4.2.10
		  Configured on: Thu Jun 18 11:00:36 CEST 2015
		  Configured by: mxe@sd-68075
		 Configure mode: production
		    Host system: i686-w64-mingw32.static
              Uname information: Linux sd-68075 3.2.0-4-amd64 #1 SMP Debian 3.2.63-2+deb7u1 x86_64 GNU/Linux
		      Libraries: 
	     Installation point: /usr/lib/mxe/usr/i686-w64-mingw32.static

Compiling Options:
------------------
               Compilation Mode: production
                     C compiler: /usr/lib/mxe/usr/bin/i686-w64-mingw32.static-gcc
                         CFLAGS:  -O2 
                       CPPFLAGS: -DH4_F77_FUNC\(name,NAME\)=NAME -DH4_BUILT_AS_STATIC_LIB=1 -DSWAP 
               Shared Libraries: no
               Static Libraries: yes
  Statically Linked Executables: no
                        LDFLAGS: 
 	 	Extra libraries: -ljpeg -lz 
 		       Archiver: i686-w64-mingw32.static-ar
 		 	 Ranlib: i686-w64-mingw32.static-ranlib

Languages:
----------
                        Fortran: no

Features:
---------
               SZIP compression: disabled
   Support for netCDF API 2.3.2: no
make -C '/usr/lib/mxe/tmp-hdf4-i686-w64-mingw32.static/hdf-4.2.10'/mfhdf/xdr -j '2' LDFLAGS=-no-undefined
make[2]: Entering directory `/usr/lib/mxe/tmp-hdf4-i686-w64-mingw32.static/hdf-4.2.10/mfhdf/xdr'
/bin/bash ../../libtool --tag=CC   --mode=compile i686-w64-mingw32.static-gcc -DHAVE_CONFIG_H -I. -I../../hdf/src -I../../hdf/src  -DH4_F77_FUNC\(name,NAME\)=NAME -DH4_BUILT_AS_STATIC_LIB=1 -DSWAP   -O2  -MT xdr.lo -MD -MP -MF .deps/xdr.Tpo -c -o xdr.lo xdr.c
/bin/bash ../../libtool --tag=CC   --mode=compile i686-w64-mingw32.static-gcc -DHAVE_CONFIG_H -I. -I../../hdf/src -I../../hdf/src  -DH4_F77_FUNC\(name,NAME\)=NAME -DH4_BUILT_AS_STATIC_LIB=1 -DSWAP   -O2  -MT xdrfloat.lo -MD -MP -MF .deps/xdrfloat.Tpo -c -o xdrfloat.lo xdrfloat.c
libtool: compile:  i686-w64-mingw32.static-gcc -DHAVE_CONFIG_H -I. -I../../hdf/src -I../../hdf/src "-DH4_F77_FUNC(name,NAME)=NAME" -DH4_BUILT_AS_STATIC_LIB=1 -DSWAP -O2 -MT xdr.lo -MD -MP -MF .deps/xdr.Tpo -c xdr.c -o xdr.o
libtool: compile:  i686-w64-mingw32.static-gcc -DHAVE_CONFIG_H -I. -I../../hdf/src -I../../hdf/src "-DH4_F77_FUNC(name,NAME)=NAME" -DH4_BUILT_AS_STATIC_LIB=1 -DSWAP -O2 -MT xdrfloat.lo -MD -MP -MF .deps/xdrfloat.Tpo -c xdrfloat.c -o xdrfloat.o
xdr.c: In function 'xdr_opaque':
xdr.c:309:9: warning: type defaults to 'int' in declaration of 'crud' [-Wimplicit-int]
  static crud[BYTES_PER_XDR_UNIT];
         ^
mv -f .deps/xdrfloat.Tpo .deps/xdrfloat.Plo
/bin/bash ../../libtool --tag=CC   --mode=compile i686-w64-mingw32.static-gcc -DHAVE_CONFIG_H -I. -I../../hdf/src -I../../hdf/src  -DH4_F77_FUNC\(name,NAME\)=NAME -DH4_BUILT_AS_STATIC_LIB=1 -DSWAP   -O2  -MT xdrstdio.lo -MD -MP -MF .deps/xdrstdio.Tpo -c -o xdrstdio.lo xdrstdio.c
libtool: compile:  i686-w64-mingw32.static-gcc -DHAVE_CONFIG_H -I. -I../../hdf/src -I../../hdf/src "-DH4_F77_FUNC(name,NAME)=NAME" -DH4_BUILT_AS_STATIC_LIB=1 -DSWAP -O2 -MT xdrstdio.lo -MD -MP -MF .deps/xdrstdio.Tpo -c xdrstdio.c -o xdrstdio.o
mv -f .deps/xdr.Tpo .deps/xdr.Plo
/bin/bash ../../libtool --tag=CC   --mode=compile i686-w64-mingw32.static-gcc -DHAVE_CONFIG_H -I. -I../../hdf/src -I../../hdf/src  -DH4_F77_FUNC\(name,NAME\)=NAME -DH4_BUILT_AS_STATIC_LIB=1 -DSWAP   -O2  -MT xdrarray.lo -MD -MP -MF .deps/xdrarray.Tpo -c -o xdrarray.lo xdrarray.c
libtool: compile:  i686-w64-mingw32.static-gcc -DHAVE_CONFIG_H -I. -I../../hdf/src -I../../hdf/src "-DH4_F77_FUNC(name,NAME)=NAME" -DH4_BUILT_AS_STATIC_LIB=1 -DSWAP -O2 -MT xdrarray.lo -MD -MP -MF .deps/xdrarray.Tpo -c xdrarray.c -o xdrarray.o
mv -f .deps/xdrarray.Tpo .deps/xdrarray.Plo
i686-w64-mingw32.static-gcc -DHAVE_CONFIG_H -I. -I../../hdf/src -I../../hdf/src  -DH4_F77_FUNC\(name,NAME\)=NAME -DH4_BUILT_AS_STATIC_LIB=1 -DSWAP   -O2  -MT xdrtest.o -MD -MP -MF .deps/xdrtest.Tpo -c -o xdrtest.o xdrtest.c
mv -f .deps/xdrtest.Tpo .deps/xdrtest.Po
/bin/bash ../../libtool --tag=CC   --mode=link i686-w64-mingw32.static-gcc  -O2   -no-undefined -o xdrtest.exe xdrtest.o  -ljpeg -lz 
libtool: link: i686-w64-mingw32.static-gcc -O2 -o xdrtest.exe xdrtest.o  /usr/lib/mxe/usr/i686-w64-mingw32.static/lib/libjpeg.a -lz
xdrtest.o:xdrtest.c:(.text.startup+0xa7): undefined reference to `xdrstdio_create'
xdrtest.o:xdrtest.c:(.text.startup+0xdc): undefined reference to `xdr_opaque'
xdrtest.o:xdrtest.c:(.text.startup+0x126): undefined reference to `xdr_opaque'
xdrtest.o:xdrtest.c:(.text.startup+0x136): undefined reference to `xdr_int'
xdrtest.o:xdrtest.c:(.text.startup+0x156): undefined reference to `xdr_vector'
xdrtest.o:xdrtest.c:(.text.startup+0x190): undefined reference to `xdr_u_int'
xdrtest.o:xdrtest.c:(.text.startup+0x1b0): undefined reference to `xdr_vector'
xdrtest.o:xdrtest.c:(.text.startup+0x1f2): undefined reference to `xdr_long'
xdrtest.o:xdrtest.c:(.text.startup+0x22c): undefined reference to `xdr_long'
xdrtest.o:xdrtest.c:(.text.startup+0x24c): undefined reference to `xdr_vector'
xdrtest.o:xdrtest.c:(.text.startup+0x286): undefined reference to `xdr_u_long'
xdrtest.o:xdrtest.c:(.text.startup+0x2a6): undefined reference to `xdr_vector'
xdrtest.o:xdrtest.c:(.text.startup+0x2e0): undefined reference to `xdr_float'
xdrtest.o:xdrtest.c:(.text.startup+0x300): undefined reference to `xdr_vector'
xdrtest.o:xdrtest.c:(.text.startup+0x33a): undefined reference to `xdr_double'
xdrtest.o:xdrtest.c:(.text.startup+0x35a): undefined reference to `xdr_vector'
xdrtest.o:xdrtest.c:(.text.startup+0x377): undefined reference to `xdr_enum'
xdrtest.o:xdrtest.c:(.text.startup+0x397): undefined reference to `xdr_vector'
xdrtest.o:xdrtest.c:(.text.startup+0x401): undefined reference to `xdrstdio_create'
xdrtest.o:xdrtest.c:(.text.startup+0x439): undefined reference to `xdr_opaque'
xdrtest.o:xdrtest.c:(.text.startup+0x49b): undefined reference to `xdr_opaque'
xdrtest.o:xdrtest.c:(.text.startup+0x50d): undefined reference to `xdr_int'
xdrtest.o:xdrtest.c:(.text.startup+0x529): undefined reference to `xdr_vector'
xdrtest.o:xdrtest.c:(.text.startup+0x5cb): undefined reference to `xdr_u_int'
xdrtest.o:xdrtest.c:(.text.startup+0x5e7): undefined reference to `xdr_vector'
xdrtest.o:xdrtest.c:(.text.startup+0x68d): undefined reference to `xdr_long'
xdrtest.o:xdrtest.c:(.text.startup+0x6e3): undefined reference to `xdr_long'
xdrtest.o:xdrtest.c:(.text.startup+0x6ff): undefined reference to `xdr_vector'
xdrtest.o:xdrtest.c:(.text.startup+0x7a1): undefined reference to `xdr_u_long'
xdrtest.o:xdrtest.c:(.text.startup+0x7bd): undefined reference to `xdr_vector'
xdrtest.o:xdrtest.c:(.text.startup+0x862): undefined reference to `xdr_float'
xdrtest.o:xdrtest.c:(.text.startup+0x87e): undefined reference to `xdr_vector'
xdrtest.o:xdrtest.c:(.text.startup+0x941): undefined reference to `xdr_double'
xdrtest.o:xdrtest.c:(.text.startup+0x95d): undefined reference to `xdr_vector'
xdrtest.o:xdrtest.c:(.text.startup+0xa03): undefined reference to `xdr_enum'
xdrtest.o:xdrtest.c:(.text.startup+0xa1f): undefined reference to `xdr_vector'
collect2: error: ld returned 1 exit status
make[2]: *** [xdrtest.exe] Error 1
make[2]: *** Waiting for unfinished jobs....
mv -f .deps/xdrstdio.Tpo .deps/xdrstdio.Plo
make[2]: Leaving directory `/usr/lib/mxe/tmp-hdf4-i686-w64-mingw32.static/hdf-4.2.10/mfhdf/xdr'
make[1]: *** [build-only-hdf4_i686-w64-mingw32.static] Error 2
make[1]: Leaving directory `/usr/lib/mxe'

real	0m14.453s
user	0m8.637s
sys	0m0.812s
```

**Comment by luckyhacky** [comment]
Made a fix two days ago.
=> https://github.com/mxe/mxe/pull/713

**Comment by luckyhacky** [comment]
manually exchange line 30

-	        CPPFLAGS="-DH4_F77_FUNC\(name,NAME\)=NAME -DH4_BUILT_AS_STATIC_LIB=1")
+	        LIBS="-lportablexdr -lws2_32" CPPFLAGS="-DH4_F77_FUNC\(name,NAME\)=NAME -DH4_BUILT_AS_STATIC_LIB=1")

**Comment by TimothyGu** [comment]
Fixed in #713.
